setup.cfg

```diff
@@ -0,0 +1,26 @@
+[metadata]
+name = wgvanitykey
+version = attr:wgvanitykey.version
+description = Simple contact information manager.
+long_description = file:README.rst
+long_description_content_type = text/x-rst
+url = https://github.com/volleybus/Wireguard-Vanity-Key-Searcher
+author = volleybus
+author_email = volleybus@github
+license = TODO
+classifiers =
+    Development Status :: 4 - Beta
+#    License :: OSI Approved :: TODO
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+platforms = any
+
+[options]
+#packages =
+#    wgvanitykey
```

Comment:
For there to be a folder in `site-packages/` named wgvanitykey instead of a (colliding) file named wgvanitykey.py, these would be uncommented